config/webpack/production.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const baseConfig = require('./base');
+const webpackMerge = require('webpack-merge');
+const { optimize, NoEmitOnErrorsPlugin, DefinePlugin } = require('webpack');
```

Comment:
You might further destructure it to omit using `optimize` later on:
```js
const {
  DefinePlugin, NoEmitOnErrorsPlugin,
  optimize: { UglifyJsPlugin, AggressiveMergingPlugin }
} = require('webpack');
```
However this looks semi-ugly but this might do the trick while keeping it nice:
```js
const { DefinePlugin, NoEmitOnErrorsPlugin, optimize } = require('webpack');
const { UglifyJsPlugin, AggressiveMergingPlugin } = optimize;
```